modules/node_modules/@ncigdc/utils/generated-relay-query-parts/fragmentGenerator.js

```diff
@@ -0,0 +1,71 @@
+/* globals _, fetch */
+/* eslint prefer-const:0, no-console:0 */
+
+let getFacetType = facet => {
```

Comment:
The contents of this file is actually meant to be copied and pasted into the browser console ._.
Might not be the best place to put it but couldn't figure out where else, no import/export
I should add that comment in I thing